ci/kind/test-e2e-kind.sh

```diff
@@ -128,18 +138,41 @@ function run_test {
-      $YML_CMD --kind --encap-mode $current_mode $manifest_args | docker exec -i kind-control-plane dd of=/root/antrea-coverage.yml
+      $YML_CMD --kind --encap-mode $current_mode $manifest_args | tee antrea-coverage.yml |docker exec -i kind-control-plane dd of=/root/antrea-coverage.yml
       $FLOWAGGREGATOR_YML_CMD --coverage | docker exec -i kind-control-plane dd of=/root/flow-aggregator-coverage.yml
+      yamlFile="antrea-coverage.yml"
   else
-      $YML_CMD --kind --encap-mode $current_mode $manifest_args | docker exec -i kind-control-plane dd of=/root/antrea.yml
+      $YML_CMD --kind --encap-mode $current_mode $manifest_args | tee antrea.yaml | docker exec -i kind-control-plane dd of=/root/antrea.yml
       $FLOWAGGREGATOR_YML_CMD | docker exec -i kind-control-plane dd of=/root/flow-aggregator.yml
+      yamlFile="antrea.yml"
   fi
   sleep 1
+  # extra configmap yaml from antrea.yml
+  yamlstartmarker=`grep -n -B 3 "antrea-agent.conf: |" $yamlFile | head -n 1 | tr -d ':-'`
+  yamlendmarker=`grep -n -- "---"  $yamlFile | grep -A 1 $yamlstartmarker | tail -n 1 | tr -d ':-'`
+  startline=`echo $yamlstartmarker+1 | bc`
+  endline=`echo $yamlendmarker-1 | bc`
+  sed -n "$startline,$endline p" $yamlFile > antrea-config.yml
+  antreaconfigfile=`realpath antrea-config.yml`
+
   if $coverage; then
-      go test -v -timeout=70m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR
+    if [[ $longcasesonly = "true" && $modeirrelevant = "true" ]]; then
+        go test -v -timeout=70m antrea.io/antrea/test/e2e -run TestLong* -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR --yaml $antreaconfigfile
+    elif [[ $modeirrelevant = "true" ]]; then
+        go test -v -timeout=70m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR --skip "antreapolicy,networkpolicy" --yaml $antreaconfigfile
+    else
+        go test -v -timeout=70m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR --skip "antctl,bandwidth,basic,batch,security,supportbuddle,tls" --yaml $antreaconfigfile
+    fi
   else
-      go test -v -timeout=65m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR
+    if [[ $longcasesonly = "true" && $modeirrelevant = "true" ]]; then
+      go test -v -timeout=65m antrea.io/antrea/test/e2e -run TestLong* -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --yaml $antreaconfigfile
+    elif [[ $modeirrelevant = "true" ]]; then
+      go test -v -timeout=65m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --skip "antreapolicy,networkpolicy" --yaml $antreaconfigfile
+    else
+      go test -v -timeout=65m antrea.io/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --skip "antctl,bandwidth,basic,batch,security,supportbuddle,tls" --yaml $antreaconfigfile
```

Comment:
you are right, since skipIfNotRequired is added now, I can add a correct set to skip to run long run cases only instead of using `-run TestLong*`, will do it.

Reply:
done